api/src/pcapi/routes/backoffice_v3/finance/forms.py

```diff
@@ -0,0 +1,33 @@
+from flask_wtf import FlaskForm
+from wtforms.validators import Optional
+
+from pcapi.core.finance import models as finance_models
+from pcapi.routes.backoffice_v3 import filters
+from pcapi.routes.backoffice_v3.forms import empty as empty_forms
+from pcapi.routes.backoffice_v3.forms import fields
+
+
+not_implemented_incident_types = [
+    finance_models.IncidentType.COMMERCIAL_GESTURE,
+    finance_models.IncidentType.FRAUD,
+    finance_models.IncidentType.OFFER_PRICE_REGULATION,
+]
+
+
+class IncidentCreationForm(FlaskForm):
+    kind = fields.PCSelectField(
+        "Type d'incident",
+        choices=[
+            (opt.name, filters.format_finance_incident_type_str(opt))
+            for opt in finance_models.IncidentType
+            if opt not in not_implemented_incident_types
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏Après investigation, je pense qu'il vaut mieux laisser ça comme ça et supprimer les types d'incidents un à un de la variable `not_implemented_incident_types`. En effet, le champs option du select côté back ne prend que la valeur et le label.